.eslintrc

```diff
@@ -1,32 +1,29 @@
 {
-    "parser": "babel-eslint",
-    "plugins": [
-        "json",
-        "react"
-    ],
-    "env": {
-        "es6": true,
-        "jest": true
-    },
-    "globals": {
-        "chrome": true,
-        "clearInterval": true,
-        "clearTimeout": true,
-        "console": true,
-        "document": true,
-        "expect": true,
-        "Promise": true,
-        "setInterval": true,
-        "setTimeout": true,
-        "window": true
-    },
-    "extends": ["standard", "standard-react"],
-    "rules": {
-        "comma-dangle": ["error", "always-multiline"],
-        "object-curly-spacing": ["error", "always"],
-        "no-var": "error",
-        "quotes": ["error", "single"],
-        "react/jsx-curly-spacing": [2, {"when": "always", "allowMultiline": false}],
-        "space-before-function-paren": 0
-    }
+	"parser": "babel-eslint",
```

Comment:
why change the spacing? Usualy tab = 2 spaces

Reply:
Accidental change that happened when I was trying to get eslint to play well with prettier. Wait for fix.